### PR TITLE
Config date picker

### PR DESCRIPTION
**Pull request by RCopeland**

### CODE REVIEW

File: packages/spark-core/package.json [hunk 1]
```diff
@@ -26,6 +26,7 @@
     "chai": "^4.1.2",
     "jsdom": "^11.12.0",
     "nyc": "^11.9.0",
+    "proxyquire": "^2.1.0",
```
Comment (afebbraro) [inline packages/spark-core/package.json]: Why the new need for another dependency? Should this been in devDependencies?

File: packages/spark-core/spark-core.js [hunk 2]
```diff
@@ -29,24 +29,27 @@ import './utilities/polyfills/classListSVG';
...
-tabs();
-toggle();
-spinners();
-WideNavigation();
-masthead();
-alerts();
-textInput();
-selectInput();
+const SparkCore = ({ datePickerConfig = {} } = {}) => {
+  detectJS();
```
Comment (afebbraro) [inline packages/spark-core/spark-core.js]: If we are removing the Init fns then do we need to update the getting started developers page?

File: packages/spark-core/package.json (hunk 1 above)
Comment (RCopeland) [reply packages/spark-core/package.json]: proxyquire lets unit tests attach stubs to internal dependancies. I'm using it to have more isolated unit tests. I originally had it in devDependencies, but the linter likes it better in dependencies, not sure why.

File: packages/spark-core/spark-core.js (hunk 2 above)
Comment (RCopeland) [reply packages/spark-core/spark-core.js]: yep, will do

**Pull request merged**